corehq/apps/app_manager/views/releases.py

```diff
@@ -255,6 +256,8 @@ def release_build(request, domain, app_id, saved_app_id):
     if saved_app.copy_of != app_id:
         raise Http404
     saved_app.is_released = is_released
+    if is_released:
+        saved_app.last_released = datetime.datetime.utcnow()
```

Comment:
should we be making it `None` in case of not `is_released`?

Reply:
Good question, I'm not sure. I'll ask.

Reply:
Updated in 42b7598f2ac0dcc291bca822f1c58a195e9199c6